Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'puma'
 
 # assets
 gem 'autoprefixer-rails'
-gem 'bootstrap'
+gem 'bootstrap', '5.0.0.alpha2'
```

Comment:
According to the [bootstrap page](https://v5.getbootstrap.com/) the alpha3 has been released.